packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -47,6 +47,12 @@ export interface DropdownSlotClassNames {
 }
 
 export interface DropdownProps extends UIComponentProps<DropdownProps, DropdownState> {
+  /** The index of the currently active selected item, if dropdown has a multiple selection. */
+  activeIndex?: number
```

Comment:
Agree that this prop name may be confusing as seems to be referencing to indexes in the list. `activeSelectedIndex`?

Reply:
it's a good suggestion :) @mnajdova @kuzhelov @Bugaa92 you agree?

Reply:
I agree, it's not obvious in terms of the Dropdown component, so using more specific name helps 👍

Reply:
renamed as Sophie suggested, thanks!